### ISSUE TITLE
Update the homepage with new content & design

### ISSUE THREAD

**Issue by winslet** [hyperledger/composer#3953]
## Context

Looking at the content on the homepage it has become apparent this might be worth looking at updating for a 1.0 release. 

## Needed content on homepage

- Info on what Composer is
- Link to install dev environment
- Link to online Playground
- Link to different versions of Composer (eg. stable / latest)
- How to get involved
    - Github repo
    - Community rocket.chat link
    - Community page link
- Cookie and privacy banner
- Docs link
- Hyperledger link
- Some of the new Composer brand materials
(please add more)

## Ideas on what content could be added 

- Stronger USP statement
- What can I do with Composer?
- Why would I use this?
- What companies are using Composer?
- Do we have any user testimonials?

Myself and @ShaunLynch will be designing and building this.

**Comment by marckaraujo** [comment]
Add also Community Projects, it helps a lot new users to start use Hyperledger.
E.g. I am developing a Serverless Rest API for Hyperledger:
https://github.com/mataide/serverless-hyperledger

**Comment by marckaraujo** [comment]
@winslet, its for composer. I am still working on it.

**Comment by ronaldlong46** [comment]
@winslet 

That's an awesome project! Looking forward to it.

**Comment by ShaunLynch** [comment]
Hey all, I've got a first pass wireframe that I'm hoping can kickoff some of the discussion points. I've just updated the current layout and content and added a few things.

- Some kind of video/gif quickly demonstrating the product in use. Giving users and impression of how the software would be used

- Testimonials to bolster user trust to try out composer

![hyperledger composer homepage wireframe](https://user-images.githubusercontent.com/35451841/40064105-bb04a836-5856-11e8-96f7-2a9c2cf9bfe0.png)

Would be keen to get some feedback and also thoughts about how we order the content.
